source/backend/apimodels/Models/Concepts/File/FilePropertyModel.cs

```diff
@@ -47,6 +47,11 @@ public class FilePropertyModel : BaseConcurrentModel
         /// </summary>
         public bool? IsActive { get; set; }
 
+        /// <summary>
+        /// get/set - Flag indicating the underlying property is retired.
+        /// </summary>
+        public bool? IsRetired { get; set; }
```

Comment:
Concepts are intended to match up approximately with their API representations. FileProperties do not have the IsRetired flag, as that lives on the property itself. IsActive, on the other hand, does belong to the FileProperty relationship, so it makes sense in this location.